src/Microsoft.FeatureManagement.AspNetCore/FeatureGateEndpointFilterExtensions.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.FeatureManagement.AspNetCore
+{
+    /// <summary>
+    /// Extension methods that provide feature management integration for ASP.NET Core endpoint building.
+    /// </summary>
+    public static class FeatureGateEndpointFilterExtensions
+    {
+        /// <summary>
+        /// Adds a feature flag filter to the endpoint that controls access based on multiple feature states.
```

Comment:
With `params`, multiple is not necessarily required. I believe the summary should account for it.

```suggestion
        /// Adds a filter to the endpoint that gates access based on whether one or more features are enabled.
```